src/commands/subdomain.rs

```diff
@@ -58,13 +70,34 @@ fn subdomain_addr(account_id: &str) -> String {
     )
 }
 
-pub fn subdomain(name: &str, user: &GlobalUser, target: &Target) -> Result<(), failure::Error> {
+pub fn subdomain(
+    name: Option<&str>,
+    user: &GlobalUser,
+    target: &Target,
+) -> Result<(), failure::Error> {
     if target.account_id.is_empty() {
         failure::bail!(format!(
             "{} You must provide an account_id in your wrangler.toml before creating a subdomain!",
             emoji::WARN
         ))
     }
+    if name.is_none() {
```

Comment:
This is kind of a nit, but instead of `if name.is_none()` and then `unwrap` later, I think you can do something kinda like this

```rust
let name = if let Some(name) = name {
  name
} else {
  return match Subdomain .......
};
```

I _think_ I like that style better than `unwrap` but I'm still learning Rust :)